src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -148,58 +213,73 @@ class DetailsBody extends React.Component<Props> {
     const criticalTrigger = rule?.triggers.find(({label}) => label === 'critical');
     const warningTrigger = rule?.triggers.find(({label}) => label === 'warning');
 
-    return (
-      <RuleDetails>
-        <span>{t('Data Source')}</span>
-        <span>{rule?.dataset && DATA_SOURCE_LABELS[rule?.dataset]}</span>
-
-        <span>{t('Metric')}</span>
-        <span>{rule?.aggregate}</span>
-
-        <span>{t('Time Window')}</span>
-        <span>{rule?.timeWindow && <Duration seconds={rule?.timeWindow * 60} />}</span>
-
-        {(rule?.dataset || rule?.query) && (
-          <React.Fragment>
-            <span>{t('Filter')}</span>
-            <span>
-              {rule?.dataset && <code>{DATASET_EVENT_TYPE_FILTERS[rule.dataset]}</code>}
-              {rule?.query}
-            </span>
-          </React.Fragment>
-        )}
-
-        <span>{t('Critical Trigger')}</span>
-        <span>
-          {this.getThresholdText(
-            criticalTrigger?.alertThreshold,
-            rule?.thresholdType,
-            true
-          )}
-        </span>
+    const ownerId = rule.owner?.split(':')[1];
+    const teamActor = ownerId
```

Comment:
I think we can replace these lines because teamActor just needs to be a falsey value or the actor object

```suggestion
    const teamActor = ownerId && {type: 'team' as Actor['type'], id: ownerId, name: ''}
```